packages/qwik/src/optimizer/src/manifest.ts

```diff
@@ -264,55 +279,87 @@ export function generateManifestFromBundles(
     },
   };
 
-  for (const hook of hooks) {
-    const buildFilePath = `${hook.canonicalFilename}.${hook.extension}`;
-
-    const outputBundle = outputBundles.find((b) => {
-      return Object.keys(b.modules).find((f) => f.endsWith(buildFilePath));
-    });
-
-    if (outputBundle) {
-      const symbolName = hook.name;
-      const bundleFileName = path.basename(outputBundle.fileName);
-
-      manifest.mapping[symbolName] = bundleFileName;
-
-      manifest.symbols[symbolName] = {
-        origin: hook.origin,
-        displayName: hook.displayName,
-        canonicalFilename: hook.canonicalFilename,
-        hash: hook.hash,
-        ctxKind: hook.ctxKind,
-        ctxName: hook.ctxName,
-        captures: hook.captures,
-        parent: hook.parent,
-        loc: hook.loc,
-      };
-
-      addBundleToManifest(path, manifest, outputBundle, bundleFileName);
+  for (const [fileName, outputBundle] of Object.entries(outputBundles)) {
+    if (outputBundle.type !== 'chunk') {
+      continue;
+    }
+    const bundleFileName = path.basename(fileName);
+
+    // Rollup doesn't provide the moduleIds in the outputBundle but Vite does
+    outputBundle.moduleIds ||= Object.keys(outputBundle.modules);
+
+    // Our special @qwik-entries chunk
+    // We need to parse this to get the minified locations of the js packets
+    if (outputBundle.moduleIds.some((id) => id.endsWith(QWIK_ENTRIES_ID))) {
+      const { code } = outputBundle;
+      const match =
+        /^(?<left>.*){\s*['"]begin-imports[^:]*:[^,}]*,?(?<entries>[^}]*)['"]end-imports['"]\s*:[^}]*}(?<right>.*)/m.exec(
```

Comment:
this regexp is created inside the loop, could we extract this to a variable and move outside the loop?